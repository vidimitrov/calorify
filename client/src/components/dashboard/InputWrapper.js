import styled from 'styled-components';

const InputWrapper = styled.div`
  margin-bottom: 20px;
`;

export default InputWrapper;
