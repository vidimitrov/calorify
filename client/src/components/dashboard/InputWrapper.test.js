import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import InputWrapper from './InputWrapper';

it('Dashboard/InputWrapper component renders correctly', () => {
  const tree = renderer.create(<InputWrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});
