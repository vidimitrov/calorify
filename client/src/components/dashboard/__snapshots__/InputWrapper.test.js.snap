// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/InputWrapper component renders correctly 1`] = `
.c0 {
  margin-bottom: 20px;
}

<div
  className="c0"
/>
`;
